content/en/news/releases/1.11.x/announcing-1.11.3/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Announcing Istio 1.11.3
+linktitle: 1.11.3
+subtitle: Patch Release
+description: Istio 1.11.3 patch release.
+publishdate: 2020-07-29
+release: 1.11.3
+aliases:
+    - /news/announcing-1.11.3
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.11.2 and Istio 1.11.3
+
+{{< relnote >}}
+
+## Changes
+
+- **Updated** to allow specifying NICs that bypassing traffic capture in Istio iptables.
+  ([Issue #34753](https://github.com/istio/istio/issues/34753))
+
+- **Added** values to the Istio Gateway Helm charts for configuring annotations on the `ServiceAccount`.  Can be used to enable [IAM Roles for Service Accounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) on AWS EKS.
+
+- **Fixed** `istioctl analyze` command to not output [IST0132] message when analyzing the gateway of the virtual service.
+  ([Issue #34653](https://github.com/istio/istio/issues/34653))
+
+- **Fixed** a bug using Service's pointer address to get its instances in case that a sidecar's egress listener has port.
```

Comment:
This doesn't sound quite right and I'm not sure what the user would have actually seen that is being fixed.

Maybe: Fixed a bug using a Service's pointer address to get its instances in the case where a sidecar's egress listener has a port.